i18n/fr.yaml

```diff
@@ -121,38 +121,38 @@ spa-research:
 cppd-research:
   other: "Participez à notre recherche"
 subscribe-title:
-  other: "Abonnez-vous à notre newsletter!"
+  other: "Abonnez-vous à notre infolettre!"
 subscribe-note:
-  other: "Inscrivez-vous à notre liste de diffusion pour obtenir des informations sur le recrutement, les mises à jour de projets et les nouveaux messages du blogue de CDS."
+  other: "Inscrivez-vous à notre liste d’envoi pour recevoir de l’information sur les postes disponibles, les nouvelles de nos projets et les nouveaux billets de blogue du SNC."
 subscribe-id:
   other: "//canada.us15.list-manage.com/subscribe/post?u=729a207773f7324e217a1d945&amp;id=5fe89f4d28"
 email-address:
-  other: "Votre courriel"
+  other: "Adresse courriel"
 subscribe:
-  other: "Abonnez-vous"
+  other: "S’inscrire"
 view-privacy-notice:
-  other: "Voir l'avis de confidentialité"
+  other: "Voir l’Avis de confidentialité"
 privacy-notice:
   other: "Énoncé de confidentialité"
```

Comment:
line 136: title should be "Avis de confidentialité", so it is the same words as in line 134.